### DISCUSSION TITLE
Xournal++ 1.2.3

### DISCUSSION THREAD

**Discussion by rolandlo** [xournalpp/xournalpp#5531] [General]
This is a new minor version of Xournal++ with bug fixes and improvements from the community.

  * Fixed background color and truncated text in LaTeX tool
  * Fixed zoom jumping and zoom steps
  * Fixed various segfaults and rendering issues
  * Fixed pressure values and project URL when saving documents
  * Enabled font hint metrics in PDF export to improve text alignment
 
See the [changelog](https://github.com/xournalpp/xournalpp/blob/release-1.2/CHANGELOG.md) for a full list of the changes.

## Changes:

* c6970f47c5204c0a81d45d3cf5d30f8b7ae8e3a6 Release 1.2.3
* 8ce8e5913382f835db332eda67b3a23c056e9628 Update changelog for 1.2.3
* a87d923ebf8d645b1d529a16054b359cba56c977 Add gtk3-immodules for generating AppImages.
* b5659f68653f4b7c987babf82390f5428dac1309 Enable font hint metrics in PDF export
* 1b454d34d67a8bdd9883f3de840df36de0611dd8 Do not export Pdf outline when only exporting part of the document
* 853b0e6f2fb168727e434a0e6367b14d5a010782 Fix toolbar management Segfault and out of date menu (#5510)
* 093c2b2c62bc3f0a54cb189f2ca06d264303111e Fix *.pdf filter for ExportAsPdf dialog (#5490)
* 26cb407e3c0e4b1969f418205a130810be202cd4 Avoid using fs::equivalent on empty path
* 119698b3cd25968065cad97c31554c5363a94668 Add and use MissingPdfUndoAction
* d928d934d18daed96a0b6346b36f0a93a46eaccd Refactor missing PDF replacement
<details><summary><b>See More</b></summary>

* e5e00df6f21ca411d69d65e70e2a7de4f30d5551 Fix debian/rules indentation
* c5adca279be59465ba1b6749c9d3df76816a2fc1 Fix PdfPagesDialog OK button sensitivity
* 592b2763e944fb5401771b636593d6efe8e24f6b Fix wrongly sized loading pages
* ac00297cc6e4f30c76d0a9fb81e7e2a6ba6ceea7 Ensure valid repaint range in VerticalToolView
* 7c315dd874e161f181439f44d14bfb2b252f3be5 Ensure non-empty repaint range in SearchResultView
* bd167e29c9d48a16fc4c48c28c688e98ecc04124 Add error checks for saving operations
* a90a7be85bdd9f740e2082c57806acf3350db1b7 Fix stroke replacement
* a5497b1892b53018f4ef990e8771f13990a154cf Ensure repainted region is non-empty in PdfElemSelection
* 1cbc67fe9531fca871b347c99d48b9b0bad6f7e0 Make .deb package build in RelWithDebInfo
* 0d97d7145fce1aab0d9bf95ff27d00d77576ede4 Improve backtrace
* 77e296c9851c7af481284eef8989d558d3ce420c Fix PROJECT_HOMEPAGE_URL
* 16e9778da965d7e6f60518c151ab83639958eba0 Allow pulling spline tangents outside of page (#5298)
* d2f5b34d8a9a73a8efcce80c400addcd27f0efe2 Fix window maximized callback and save setting
* c899cec918f2d92454cce6282327aa0536b67ebd Use Util::rgb_to_hex_string everywhere
* dc4b82d29aa2a0a5cadd21726a27fd2cba905b25 Fix Util::rgb_to_hex_string
* 7d0f9283f61ebf81b13182c85e7b5eb6f0c64691 Fix long text being truncated in LaTeX tool
* eb3c223ac05e7b9b70401a8f6afcbaa5db303a58 Identify Touchpad and Trackpoint as same device
* 178b4aeb8a86757e7231d1f5f3aacfe24bbc5057 Fix zoom centers and coordinate conversions
* fc1340efe5391a30aeb2ea76f9c5163b175db604 Apply review suggestions
* 7b8ba108afa48a26fca87fed08fcf0c145bfb0c6 fix zoom jumping and zoom steps
* 734ead8017401d962a5fe73a9a23c2476a334b1e Improve arrow shape with respect to fill
* d763bffa521e88f2dab59b4d2a49f43696e99eea fix redundant negative pressure.
* d8da6c899bd626c9d744956bed187cd16f0f3c50 Fix Spline live segment display
* ee03a0e7c3491121bb17c48d0728fb181f5fa8c8 Automated version bump to 1.2.3~dev

This list of changes was [auto generated](https://dev.azure.com/xournalpp/ce85de88-4b71-44a0-8a57-2707c0b9a6e7/_release?releaseId=808&_a=release-summary).</details>

<hr /><em>This discussion was created from the release <a href='https://github.com/xournalpp/xournalpp/releases/tag/v1.2.3'>Xournal++ 1.2.3</a>.</em>

**Comment by gomarcus** [comment]
Hey, thanks so much for all the great work!
Testing out the new version on windows 11 (10.0.22631 Build 22631), I happened to notice that not all jumping around of the canvas is fixed. While zooming got so much better, horizontal movement/navigation on a high zoom level has quite a bit jumping/jitter left. The problem seems to worsen when performing multiple left/right motions ... multiple times without touch-up (touch display).
